test/integration/base_integration_test.h

```diff
@@ -167,7 +167,7 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
   envoy::config::core::v3::Node last_node_;
 
   // Functions for testing reloadable config (xDS)
-  void createXdsUpstream();
+  virtual void createXdsUpstream();
```

Comment:
will this break other tests?

Reply:
No. Until now other tests just called this method. The new test is the only one overriding it. In c++ when a method is not declared virtual it cannot be overridden.